modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js

```diff
@@ -579,6 +807,47 @@ SideNavigation.prototype = {
 		}
 
 		INSTANCE_MAP.delete(instance.toggler);
+
+		if (instance._handleClosedStart) {
+			document.removeEventListener(
+				'closedStart.lexicon.sidenav',
+				instance._handleClosedStart
+			);
+		}
+
+		if (instance._handleClosed) {
+			document.removeEventListener(
+				'closed.lexicon.sidenav',
+				instance._handleClosed
+			);
+		}
+
+		if (instance._handleOpenStart) {
+			document.removeEventListener(
+				'openStart.lexicon.sidenav',
+				instance._handleOpenStart
+			);
+		}
+
+		if (instance._handleOpen) {
+			document.removeEventListener(
+				'open.lexicon.sidenav',
+				instance._handleOpen
+			);
+		}
+
+		if (instance._handleOnScreenChange) {
+			document.removeEventListener(
+				'screenChange.lexicon.sidenav',
+				instance._handleOnScreenChange
+			);
+		}
```

Comment:
These are all idempotent, so you could get rid of the `if` to make them a bit less bulky:

```js
document.removeEventListener(
	'closedStart.lexicon.sidenav',
	instance._handleClosedStart
);

document.removeEventListener(
	'closed.lexicon.sidenav',
	instance._handleClosed
);

// etc
```